server/api/api.raml

```diff
@@ -663,6 +667,41 @@ types:
       500:
         description: PD server failed to proceed the request.
 
+  /limit:
+    description: The all stores' balance rate limit.

```

Comment:
`The stores` or `All stores`.

Reply:
Suggest using: balance rate limit for all stores.